### PR TITLE
Add append plugin

### PR DESCRIPTION
**Pull request by devoxel**

### CODE REVIEW

File: steely/plugins/append.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'''append some text to some other text'''
+
+__author__ = 'devoxel'
+COMMAND = 'append'
+
+def appendify(s, to_append):
+    o = a + b
```
Comment (byxor) [inline steely/plugins/append.py]: Where are `a` and `b` defined?

File: steely/plugins/append.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    if o > 300:
+        return "a dollop of spam"
+    return o
+
+def main(bot, author_id, to_append, thread_id, thread_type, **kwargs):
+    message = bot.fetchThreadMessages(thread_id=thread_id, limit=2)[1]
```
Comment (byxor) [inline steely/plugins/append.py]: nitpick: `s/message/last_message`
nitpick: `s/to_append/extra_text`

File: steely/plugins/append.py (hunk 1 above)
Comment (CianLR) [reply steely/plugins/append.py]: Lmao aaron well done

Comment (devoxel) [reply steely/plugins/append.py]: oops, fixed in next commit

File: steely/plugins/append.py (hunk 2 above)
Comment (devoxel) [reply steely/plugins/append.py]: Done

**Pull request merged**